app/test_main.py

```diff
@@ -1,3 +1,13 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_animal_not_first_year() -> None:
+    assert get_human_age(14, 12) == [0, 0]
+
+
+def test_animal_one_years() -> None:
+    assert get_human_age(15, 23) == [1, 1]
+
+
+def test_animal_two_years() -> None:
+    assert get_human_age(28, 28) == [3, 2]
```

Comment:
* Make sure you've tests checks all edged situations such as:
  * if output should change with some integer value you should check that it is not changed with previous value;
  * if function resive data out of normal range, such as negative numbers, zero, or realy large numbers;
  * if the function receives an incorrect type of data it raises the correct exception.

* Use `@pytest.mark.parametrize`